src/main/kotlin/ro/hibyte/betting/service/BetTypeService.kt

```diff
@@ -0,0 +1,52 @@
+package ro.hibyte.betting.service
+
+import org.springframework.stereotype.Service
+import ro.hibyte.betting.dto.CompleteBetTypeDto
+import ro.hibyte.betting.entity.BetTemplate
+import ro.hibyte.betting.entity.BetType
+import ro.hibyte.betting.exceptions.types.BetTypeNotFoundException
+import ro.hibyte.betting.repository.BetTemplateRepository
+import ro.hibyte.betting.repository.BetTypeRepository
+
+@Service
+class BetTypeService(private val betTypeRepository: BetTypeRepository,
```

Comment:
For multiple line constructor parameters/method parameters please do this (remember the trailing comma):

```kotlin
class BetTypeService(
    private val betTypeRepository: BetTypeRepository,
    private val betTemplateRepository: BetTemplateRepository,
) {
```

This is better, because it helps with preventing unnecessary merge conflicts in the future

Reply:
Done.